manifests/a/Amazon/Games/3.0.9201.1/Amazon.Games.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Amazon.Games
+PackageVersion: 3.0.9201.1
+InstallerLocale: en-US
+InstallerType: exe
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+UpgradeBehavior: install
+ReleaseDate: 2024-06-06
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.amazongames.com/AmazonGamesSetup.exe

```

Comment:
It seems this package uses vanity URL, which always point to the latest version.

Also it seems the `InstallerUrl` is just an online web installer.

![image](https://github.com/user-attachments/assets/ea88497e-3597-4cea-8fe9-5de58d04939b)

Reply:
Vanity URLs are okay and we have many of them. Our process for handling their package updates could be improved. 

Our automation recently identified this package as having a duplicate hash as another package, so it would appear that this is just modifying the package number of a package already in the repo. 
- Is this like a nested installer? Or is it like an installer that only adds shortcuts to websites? Is there a change to the manifest that could be made to resolve this issue? 

I'm going to approve this PR, to complete this set of tasks. But please open a PR with any modifications that you feel would improve the package's manifest.